Ocarina/OcarinaManager.swift

```diff
@@ -173,6 +181,14 @@ public class OcarinaManager: NSObject {
 
 extension OcarinaManager: URLSessionDataDelegate {
     
+    public func urlSession(_ session: URLSession,
```

Comment:
Small style thing here: Could you make the function name and parameters all on one line, like all other delegate methods :)